Dockerfile

```diff
@@ -0,0 +1,19 @@
+# Copyright 2017, 2019 the Velero contributors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM debian:stretch-slim
```

Comment:
we don't need to do this now, but it would be nice to switch to a [multi-stage build](https://docs.docker.com/develop/develop-images/multistage-build/) here - would likely simplify some of the stuff in the Makefile as well